src/pages/ChallengeDetails/ChallengeDetails.jsx

```diff
@@ -4,20 +4,20 @@ import DashboardNavbar from "../../components/DashboardNavbar/DashboardNavbar";
 import clock from "../../assets/icons/clock.svg";
 import pen from "../../assets/icons/pen.svg";
 import bulb from "../../assets/icons/lightbulb.svg";
-import challenges from "./challenges.json";
+import { mockChallenges } from "../../mock-data/mockChallenges";
 
 export function ChallengeDetails() {
 
     const {challengeId} = useParams()
 
-    const challengeData = challenges.challenges[challengeId]
+    const challengeData = mockChallenges.challenges[challengeId - 1]
```

Comment:
ChallengeId will never be a string? Cause this would break. I rather you search the mockdata for the id instead of this workaround

Reply:
I've updated this to use to find the challenge based on its id